composer.json

```diff
@@ -14,7 +14,7 @@
     }
   ],
   "require": {
-    "php": "^7.2",
+    "php": "^7.2 || ^8.0",
```

Comment:
dat was dus alles? 😄 
ik denk wel dat we een PR bij hun ook moeten doen want ik zie dat de repo recente commits heeft. dus als we lang op een fork blijven raken we out of date met swagger.

Reply:
Ja ik ga wel even een PR indienen. Ik heb wat wijzigingen uit hun eigen PR voor PHP8 overgenomen

Reply:
IK laat deze branch voorlopig nog even bestaan. Dan is het duidelijk dat we nog niet een definitieve oplossing hebben